.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'Please replace with a clear and descriptive title'
+labels: 'type: bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
Should we have something else here, since this for a CLI tool?

Reply:
Good point, fixed it here: https://github.com/netlify/cli/pull/1090/files#diff-74c2649c999338ed7a2833d79d4b25c3R18